addons/pkg/crdwait/crd_wait_test.go

```diff
@@ -182,29 +250,7 @@ var _ = Describe("WaitForCRDs", func() {
 			crdWaiter.PollTimeout = time.Second * 2
 
 			Expect(crdWaiter.WaitForCRDs(crds, pod, "foo")).To(HaveOccurred())
-			Expect(<-fakeRecorder.Events).To(ContainSubstring(fmt.Sprintf("The GroupVersion '%s' is not available yet", clusterapiv1beta1.GroupVersion.String())))
-
-		})
-	})
-
-	Context("when GroupVersion exists but not all api-resources exist ", func() {
```

Comment:
Is it expected that this test is removed ?

Reply:
Yes, cause there wasn't a way I could test APIReader.Get()returning an error other than NotFound. The function is not mocked in the test file